App.js

```diff
@@ -17,15 +18,18 @@ import { WebView } from 'react-native-webview';
 class App extends Component {
   
   render() {
-    const runFirst = `
+     const runFirst = `
+      document.body.style.backgroundColor = 'red';
```

Comment:
why do we have this color set ?

Reply:
ahh we don't , It is just left from my testing ...